.drone.yml

```diff
@@ -2,7 +2,8 @@ pipeline:
   docker:
     image: plugins/docker
     secrets: [ docker_username, docker_password ]
-    repo: lucap/drone-now
-    auto_tag: true
+    repo: andyet/drone-now
```

Comment:
Could you put the lucap/drone-now repo for docker image ?

Reply:
sure that is fixed in the latest commit. I will have to switch back after merging though since that is my own drone config file.